index.js

```diff
@@ -283,6 +289,12 @@ function choosePluralForm(text, locale, count) {
   return text;
 }
 
+// Determines if an object passes as a smart_count parameter for
+// pluralization is valid
+function isValidSmartObject(count) {
+  return (typeof count.count === 'number' && !!count.value);
```

Comment:
there's no need for these parens. also, what are you testing with `!!count.value`? this will a) fail with zero, b) pass with ±Infinity, c) isn't very clear.